build.rs

```diff
@@ -0,0 +1,116 @@
+#[macro_use]
+extern crate lazy_static;
+
+use std::fmt;
+use std::path::Path;
+use std::fs::{File, read_dir, metadata};
+use std::io::{Write, BufWriter};
+
+// Location of the dist folder
+#[cfg(not(target_os = "windows"))]
+const DIST: &str = "frontend/dist/";
+#[cfg(target_os = "windows")]
+const DIST: &str = "frontend\\dist\\";
```

Comment:
`r"Raw\strings\are\your\friend"` :) (Can Windows _really_ not deal with forward slashes here?)